### ISSUE TITLE
es4x.d.ts in conflict with a existing npm package "util"

### ISSUE THREAD

**Issue by outloudvi** [reactiverse/es4x#350]
tl;dr: `es4x.d.ts` has the declaration of a module called `util`, which collides with a famous npm module [`util`](https://www.npmjs.com/package/util), causing errors sometimes.

Is it a problem on `es4x`'s side, or it's my using something wrongly?

---
The long story:

I want to use es4x with Typescript as well as Webpack (for both backend and frontend). To make TypeScript recognize `vertx`, I need to:

``` typescript
/// <reference types="es4x" />
```

and include the declarations. In the declaration there is a module called `util`:

https://github.com/reactiverse/es4x/blob/71d6413be180c1f51a9c0f0d879ec7af9cdc33e7/pm/src/main/resources/META-INF/es4x-commands/es4x.d.ts#L166

However, `webpack` is depending on a package, also called [`util`](https://www.npmjs.com/package/util):

```
$ npm ls util
es4x-example@1.0.0 /tmp/es4x-example
└─┬ webpack@4.43.0
  └─┬ node-libs-browser@2.2.1
    ├─┬ assert@1.5.0
    │ └── util@0.10.3 
    └── util@0.11.1 
```

which creates an error for TypeScript, saying:

```
node_modules/@types/es4x.d.ts:166:16 - error TS2665: Invalid module name in augmentation. Module 'util' resolves to an untyped module at '/tmp/es4x-example/node_modules/util/util.js', which cannot be augmented.

166 declare module "util" {
                   ~~~~~~
```

**Comment by outloudvi** [comment]
Hi. I find that many Node.js builtin modules (such as "buffer" and "net") are not implemented in the es4x environment, so maybe I'm radically wrong attempting to use other npm modules with es4x (because many of them are designed for Node.js and invoke Node.js builtin modules). Given this condition, this issue might not be a bug.

**Comment by pmlopes** [comment]
@outloudvi yes, think of `es4x` as `deno`. We can handle javascript, we can handle npm modules, but not really node specific features.

**Comment by pmlopes** [comment]
As with vert.x we have fully promissified the APIs, all async calls (from vert.x will work with `async/await` so when a release of 4.0 happens I'll remove the "utils" module as it is not useful anymore and this issue will be solve by it's own.

**Comment by pmlopes** [comment]
Depends on https://github.com/vert-x3/issues/issues/500

**Comment by outloudvi** [comment]
Oh, I see that! Thanks for your explanation!
